awesome_cart/awc.py

```diff
@@ -745,6 +747,7 @@ def sync_awc_and_quotation(awc_session, quotation, quotation_is_dirty=False, sav
 	call_awc_sync_hook(awc_session, quotation)
 
 	save_and_commit_quotation(quotation, quotation_is_dirty, awc_session, commit=True, save_session=awc_is_dirty)
+	#calculate_quotation_discount(quotation, awc_session)
```

Comment:
Déjà vu. 😅

Reply:
Not the same as `calculate_tax_and_totals` btw

Reply:
haha, yeah not the same. I moved that into the save_and_commit_quotation() method so its ubiquitous during calcs.